### ISSUE TITLE
Only downloading first image of a post

### ISSUE THREAD

**Issue by CaolyX9** [rarcega/instagram-scraper#446]
When scraping a user's post, the program only downloads the first image of the post, not every image of a single post. Is this normal ? How can i download every images of a post ?

**Comment by danielo93** [comment]
What do you mean for "post"? If you want to scrap every images of a profile, you have to code:

instagram-scraper <username> -u <your username> -p <your password>  -t image

I don't know if I've understood right your answer.

**Comment by Hrxn** [comment]
Some posts only have one image (like old IG), other posts have multiple images.
This is not some new feature by the way, IG now had this a long time.

**Comment by CaolyX9** [comment]
Yeah like @Hrxn said, some post have multiple photos, so you can swipe on a post to see all of it's photos, but the scrapper only downloads the first pic of a post

**Comment by danielo93** [comment]
Uh ok, now I catch the problem. Sorry for my misunderstood

**Comment by CaolyX9** [comment]
Does it do the same thing for you guys ?

**Comment by ekalin** [comment]
Are you using the --template (or -T) option to set a custom file name template? If so, you must include {urlname}, because that's the only thing that differes between each post of a multi-image post.

**Comment by CaolyX9** [comment]
Im using this : -T={username}_{year}-{month}-{day}_{h}-{m}-{s}

**Comment by ekalin** [comment]
As per my previous post, include {urlname} in your template.

**Comment by val-storm** [comment]
Anyone have any luck with this yet?
Including {urlname} in argument to T does not download multiple page posts for me.
